gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageImplTest.java

```diff
@@ -226,6 +226,11 @@ public void conflictingWrites_noOverwrite_lastFails() throws IOException {
     StorageResourceId resourceId = new StorageResourceId(TEST_BUCKET, name.getMethodName());
     TrackingStorageWrapper<GoogleCloudStorageImpl> trackingGcs =
         newTrackingGoogleCloudStorage(GCS_OPTIONS);
+    // Have separate request tracker for channels as clubbing them into one will cause flakiness
+    // while asserting the order or requests.
```

Comment:
To assert order of the requests you will need to add `.inOrder();` call after assert.

Reply:
ohh, added.